src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -139,3 +147,12 @@ interface EnforceQueryPipelineConfigurationModel {
 interface ImpersonationRestrictionsModel {
     allowedUserIds: QueryAuthenticationModel[];
 }
+
+export enum ApiKeyStatus {
```

Comment:
This should probably go in the Enums.ts file

Reply:
Perfect going to export it to that file